### DISCUSSION TITLE
LXMF <> IPFS?

### DISCUSSION THREAD

**Discussion by samuk** [markqvist/Reticulum#45] [Ideas]
It seems that LXMF/ Reticulum originate in a very offline-first way.

I wonder if you'd given much thought to bridges or syncing with other global systems such as IPFS?

If LXMF messages additionally existed in IPFS then they'd be readable globally without relying on a traditional client<>server relationship?

https://www.theverge.com/2021/1/19/22238334/brave-browser-ipfs-peer-to-peer-decentralized-transfer-protocol-http-nodes

**Comment by markqvist** [comment]
I am definitely going to look into this as soon as I can prioritise it. There might be some very interesting possibilities.
